test/suite/hook_tiered.py

```diff
@@ -34,16 +34,24 @@
 #
 # Substitute tiered tables for regular (row-store) tables in Python tests.
 #
-# These hooks can be used to run the existing cursor tests on tiered tables.
-# They identify tests that create row-store tables and create tiered tables
-# instead.  The hook takes an optional argument to specify how many tiers
-# to create. The default is 2.
+# These hooks are intended for basic testing of tiered tables.  There are several
+# pieces of functionality here.
 #
-# To run with 3 tiers per table:
-#     ../test/suite/run.py --hooks tiered=3 cursor
+# 1. We add tiered storage parameters to the config when calling wiredtiger_open()
+#
+# 2. If we create an object that is *not* a table:, we add options to its config
+#    so that it will be stored local-only.  Tiered storage isn't intended (yet?) for
+#    use with LSM or colstore.
+#
+# 3. We add a call to flush_tier() after each call to checkpoint().
+#
+# 4. We stub out some functions (currently just drop) that aren't supported by tiered
+#    tables.  This will break tests of those functions.  But often when they are used
+#    in other tests, we can get away with returning success without performing the operation.
+#
+# To run, for example, the cursor tests with these hooks enabled:
+#     ../test/suite/run.py --hooks tiered cursor
```

Comment:
Please make sure something reasonable happens with `--hooks tiered tiered` too!

Reply:
At the moment the hook works with the tiered tests.  But I'll update the hook to just skip those tests, since they already have code to invoke tiered storage.